src/components/simulation/tray.tsx

```diff
@@ -33,23 +34,26 @@ export const Tray: React.FC<IProps> = (props) => {
 
   return (
     <div className={`tray ${hidden ? "hidden" : ""}`} ref={drop}>
-      <SortingTray />
-      <div className="header">
-        <div className="title">{t("SORTINGTRAY")}</div>
-        <button className="close" onClick={onHideTray} aria-label={t("BUTTON.CLOSE")}>
-          <CloseIcon />
-        </button>
+      <div className="tray-wrapper" ref={wrapperRef}>
```

Comment:
I was imagining the wrapper being outside the tray, but I doubt it makes any difference.